### PR TITLE
TechDebt101

### PR DESCRIPTION
**Pull request by faraakhatTW**
## Proposed changes

- Re-write MUI imports in `publicDataDashboard` 
- Fix failing test in clients and shared folder. 
Following TechDebt card [link]( https://github.com/PublicDataWorks/complaint-manager/issues/389)

## Types of changes

What types of changes does your code introduce to Complaint Manager?
_Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [x] Maintenance / Documentation Update

## Instance-files

- [ ] Do your changes require changes to instance-files?
  - if so please include a link to pull request for [instance-files demo](https://github.com/PublicDataWorks/instance_files_pm)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://publicdataworks.github.io/complaint-manager/technical-content/contributing.html) doc
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/client/publicDataDashboard/DashboardDataSection.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { lazy, Suspense } from "react";
-import { Grid, Link as MUILink, Typography } from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import MUILink from "@material-ui/core/Link";
```
Comment (heliofreitastw) [inline src/client/publicDataDashboard/DashboardDataSection.js:3]: Should this import have the `Link as MUILink` like it was before? Looks like `MUILink` is a name someone gave to this import

File: src/client/publicDataDashboard/DashboardAbout.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import React from "react";
-import { Grid, Icon, Link as MUILink, Typography } from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import Icon from "@material-ui/core/Icon";
+import MUILink from "@material-ui/core/Link";
```
Comment (heliofreitastw) [inline src/client/publicDataDashboard/DashboardAbout.js:4]: Same thing here with the `MUILink`.

File: src/client/policeDataManager/shared/components/TextTruncate.test.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import { mount } from "enzyme/build/index";
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 import TextTruncate from "./TextTruncate";
 import ActivityDisplay from "../../cases/CaseDetails/CaseNotes/ActivityDisplay";
 import DashboardDataSection from "../../../publicDataDashboard/DashboardDataSection";
 
+configure({ adapter: new Adapter() });
```
Comment (heliofreitastw) [inline src/client/policeDataManager/shared/components/TextTruncate.test.js:9]: what this code does?

Comment (faraakhatTW) [reply src/client/policeDataManager/shared/components/TextTruncate.test.js:9]: this is telling Enzyme to use a specific adapter, corresponding for react version we are using. Basically it help setup Enzyme adapter globally in order to run test. Before that this test were throwing errors. Here is the link to a documentation [Link](https://enzymejs.github.io/enzyme/docs/installation/)

File: src/client/publicDataDashboard/DashboardDataSection.js (hunk 1 above)
Comment (faraakhatTW) [reply src/client/publicDataDashboard/DashboardDataSection.js:3]: Its just naming convention which has no direct effect to a import itself as to Link as MUILink. MUILink serves same purpose. Its just shorter and better naming if we use different package that has export property like Link.

File: src/client/policeDataManager/shared/components/TextTruncate.test.js (hunk 3 above)
Comment (heliofreitastw) [reply src/client/policeDataManager/shared/components/TextTruncate.test.js:9]: thank you

File: src/client/publicDataDashboard/DashboardDataSection.js (hunk 1 above)
Comment (heliofreitastw) [reply src/client/publicDataDashboard/DashboardDataSection.js:3]: thank you

**Review by heliofreitastw — APPROVED** [review]

File: src/client/publicDataDashboard/DashboardGlossary.test.js [hunk 4]
```diff
@@ -4,7 +4,10 @@ import React from "react";
 import { BrowserRouter as Router } from "react-router-dom";
 import createConfiguredStore from "../createConfiguredStore";
 import { Provider } from "react-redux";
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 
+configure({ adapter: new Adapter() });
```
Comment (sbhamani) [inline src/client/publicDataDashboard/DashboardGlossary.test.js:10]: If you look at /src/setupTests.js it already configures the Enzyme adapter and we shouldn't have to re-instantiate it here. See if these are able to run w/o re-instantiating and compare to other Enzyme tests that don't explicitly configure the adapter.

**Review by sbhamani — CHANGES_REQUESTED** [review]
Check the adapter comment

**Review by sbhamani — APPROVED** [review]

**Pull request merged**